docs/user.md

```diff
@@ -1,15 +1,15 @@
-# Welcome to the OpenSARlab User Guide
+# Welcome to the OpenSARLab User Guide
 
 1. [Configuring Multi-Factor Authentication](user-guides/mfa.md)
 1. [Jupyter Notebook Intro](user-guides/how_to_run_a_notebook.md)
 1. [Running Jupyter Notebooks](user-guides/how_to_run_a_notebook.md)
 1. [Jupyter Magic Commands](user-guides/jupyter_magic.md)
-1. [OpenSARlab Account Details](user-guides/OpenSARlab_environment.md)
```

Comment:
It appears that the documents themselves were not renamed. This means that links like these will break and give a 404.